sample/Tailor/tools/bazelwrapper

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+set -e
+
+# Make sure we're in the project root directory.
+SCRIPTPATH="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" && pwd  )"
+pushd "$SCRIPTPATH/.." > /dev/null
+trap popd > /dev/null ERR EXIT
+
+# Go to bazel release page
+# These are typically posted in groups.google
+# https://groups.google.com/forum/#!forum/bazel-discuss
+BAZEL_VERSION="0.16.1"
```

Comment:
0.18.0 ?

Reply:
XCHammer is currently not tested on that version even though it does support it